.github/CODEOWNERS

```diff
@@ -146,8 +146,7 @@
 
 
 # Dev workflow team.
-# TODO(#10472): Revert ownership to @vinitamurthi after 2020-09-05.
-/.circleci/ @BenHenning @U8NWXD @kevintab95
+/.circleci/ @BenHenning @U8NWXD @kevintab95 @vinitamurthi
 /.eslintignore @bansalnitish @Hudda
```

Comment:
@vinitamurthi Wanted to confirm whether [not owning back .travis.yaml file](https://github.com/oppia/oppia/pull/10473/files#diff-5cfb934ca0ec4c1743905a51dc428b84R157) is expected? [Just wanted to confirm whether that's expected. :)]

Reply:
Ah thanks for pointing this out! That wasn't expected, added myself back to travis as well